src/components/SelectionTable/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import rows from './data';
+import columns from './columns';
+import {DataGrid, DataGridProps} from "@mui/x-data-grid";
+
+export interface historyElement {

```

Comment:
it is no point to have this interface here and just export it. Lets move this block to "src/components/SelectionTable/data.ts" file where it is used